script.js

```diff
@@ -1,55 +1,84 @@
-function erroMessage() {
-    const radioSelected = document.querySelectorAll('input[name="voto"]');
-    const option = false;
+document.addEventListener('DOMContentLoaded', () => {
+    const options = [
+        {
+            description: 'Ótimo',
+            id: 'otimo',
+        },
+        {
+            description: 'Bom, mas pode melhorar',
+            id: 'bom',
+        },
+        {
+            description: 'Não gostei',
+            id: 'nao-gostei',
+        },
+    ];
+    const container = document.getElementById('enquete-options');
 
-    radioSelected.forEach((radio) => {
-        if (radio.checked) {
-            option = true;
-        }
-    });
+    options.forEach(option => {
+        const input = document.createElement('input');
+        input.type = 'radio';
+        input.name = 'voto';
+        options.findIndex(option => option.id === 'input');
+        input.value = option.description;
+        input.setAttribute('required', option.id === 'otimo' ? 'true' : '');
 
-    if (!option) {
-        const strong = document.createElement('strong');
-        const text = document.createTextNode('Selecione uma opção antes de enviar.');
-        strong.appendChild(text);
-        strong.classList.add('has-text-danger');
-        document.querySelector('form').appendChild(strong);
-    }
-}
+        const label = document.createElement('label');
+        const section = document.createElement('section');
+        label.htmlFor = option.id;
+        label.textContent = option.description;
+        section.appendChild(input);
+        section.appendChild(label);
+        container.appendChild(section);
+        input.classList.add('mr-2');
 
-function unlockButton() {
-    const activateButton = document.querySelector('.disabled');
-    activateButton.classList.remove('disabled');
-}
+        const url = new URL(window.location.href);
```

Comment:
Deixe as declarações sempre no início do escopo, a não ser que seja algo muuuito específico que depende de outras lógicas para poder ser criada.